app.py

```diff
@@ -149,12 +157,15 @@ def table():
 
 @app.route('/delete/contacts/<id>', methods=["POST"])
 @login_required
-def delete(id):
-
-    phonebook.delete(id)
-    flash(f"ID:{id} Deleted")
+def delete_contact(id):
```

Comment:
You're probably starting to feel the need to have separate files for handlers of each model. :)

Reply:
Exactly, I was thinking of implementing the blueprint approach